### DISCUSSION TITLE
User without permission to delete tickets or followups can delete them anyways

### DISCUSSION THREAD

**Discussion by rodrigolfh** [django-helpdesk/django-helpdesk#1222] [Q&A] [answered]
I set my own user to not being able to do so like this:
![image](https://github.com/user-attachments/assets/ab46b269-f346-4441-bcc4-8d7ab49cbc98)
![image](https://github.com/user-attachments/assets/8ed68e49-54b2-4cd0-9c89-f0cf7810851c)

But my user can still delete tickets. What am I doing wrong?

Thanks in advance.

**Comment by uhurusurfa** [comment, marked as answer]
If the user is a staff member (Es Staff ✅ ) then they can delete tickets they have access to.

It would require an enhancement to the code base to make it configurable on a per user basis to control who can delete tickets.

Makingit configurable to be superuswer users only would not be difficut if you are a pythin programmer otherwise add an issue requesting an enhancement to the user permissions.
